Classes/KIFTextInputTraitsOverrides.h

```diff
@@ -0,0 +1,12 @@
+//
+//  KIFTextInputTraitsOverrides.h
+//  KIF
+//
+//  Created by Harley Cooper on 1/31/18.
+//
+
+#import <Foundation/Foundation.h>
+
+FOUNDATION_EXTERN void KIFSetAutocorrect(BOOL);
```

Comment:
Might be more clear to name these `KIFForceDisableAutocorrect`?

Reply:
Same goes for the other two as well.

Reply:
Nevermind, I see what you're doing (as it is disabled by default)

Reply:
Maybe `KIFAllowDefaultAutocorrectBehavior()` might be more clear, but it is definitely a nit.